api/src/main/java/org/openmrs/module/m2sysbiometrics/service/impl/NationalRegistrationFailureServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package org.openmrs.module.m2sysbiometrics.service.impl;
+
+import org.openmrs.api.impl.BaseOpenmrsService;
+import org.openmrs.module.m2sysbiometrics.dao.M2SysNationalRegistrationFailureDao;
+import org.openmrs.module.m2sysbiometrics.model.NationalRegistrationFailure;
+import org.openmrs.module.m2sysbiometrics.service.NationalRegistrationFailureService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class NationalRegistrationFailureServiceImpl extends BaseOpenmrsService
+        implements NationalRegistrationFailureService {
+
+    @Autowired
+    private M2SysNationalRegistrationFailureDao dao;
+
+    @Override
+    public NationalRegistrationFailure save(NationalRegistrationFailure nationalRegistrationFailure) {
+        dao.save(nationalRegistrationFailure);
```

Comment:
shouldn't the saved object be returned? 
M2SysNationalRegistrationFailureDao.save() returns NationalRegistrationFailure